Components/Linux/wordpress-recipes-linux/README.md

```diff
@@ -0,0 +1,46 @@
+# WordPress ImageBuilder Components
+
+## Variants
+### install_wordpress.yml
+
+1. Assumes php 7.4 is/will be pre-installed (eg from the Amazon-managed `php-7-4-linux` ImageBuilder component)
```

Comment:
I think we should make a statement about the underlying Operating System these are designed for. I am presuming that would be Amazon Linux 2, but we should state this as the samples (using `yum`) clearly won't work on other platforms such as Ubuntu. (I'm not saying we need to make them work elsewhere, just to state what the samples are intended to work on)